frontend/src/ButtonMenu.js

```diff
@@ -0,0 +1,18 @@
+import React, {useState} from 'react';
+import { Menu, Dot } from './ButtonMenu.styled';
+
+const ButtonMenu = (props) => {
+  const [open, setOpen] = useState(false);
```

Comment:
jeśli piszemy jakąś zmienną typu boolean to dodaj na począstku `is`. W tym przypadku `isOpened`